packages/concerto-core/test/serializer.js

```diff
@@ -78,6 +78,11 @@ describe('Serializer', () => {
         o String newValue
         }
 
+        concept Dictionaries {
+            o Record<String, String> simpleDictionary optional
+            // o Record<String, SampleParticipant> complexDictionary optional
```

Comment:
Note that Set<ComplexType> requires identity for ComplexType, so should only be supported for Identifiables.